features/keystone/identity/ldap.pan

```diff
@@ -9,18 +9,14 @@ include 'components/metaconfig/config';
 prefix '/software/components/metaconfig/services/{/etc/keystone/keystone.conf}';
 'contents/identity/domain_specific_drivers_enabled' = 'True';
 'contents/identity/domain_config_dir' = '/etc/keystone/domains';
-'contents/identity/driver' = 'ldap';
+'contents/identity/driver' = 'sql';
 'contents/resource/driver' = 'sql';
 'contents/assignment/driver' = 'sql';
 
 # All ldap configuration is put on /etc/keystone/domains/keystone.DOMAIN_NAME.conf
 prefix '/software/components/metaconfig';
 'services' = {
-  # default domain is still on SQL
```

Comment:
It is only mandatory when the default identity driver in /etc/keystone/keystone.conf is not set to SQL. If the default is not set to SQL and you try to add a third domain in the same way as the default then you get problems.